app_auth_test.py

```
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+"""
+認証システムのテスト用アプリケーション
+最小限の機能で認証システムをテスト
+"""
+from flask import Flask, render_template, request, redirect, url_for
+from flask_login import LoginManager, login_user, logout_user, login_required, current_user
+from flask_bcrypt import Bcrypt
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import DataRequired, Email, EqualTo, Length, ValidationError
```

Ruff (0.12.2)

`wtforms.validators.Email` imported but unused

Remove unused import

(F401)